src/components/safe-apps/SafeAppLandingPage/index.tsx

```diff
@@ -30,17 +30,19 @@ const SafeAppLanding = ({ appUrl, chain }: Props) => {
   const showDemo = chain.chainId === CHAIN_ID_WITH_A_DEMO || !!backendApp?.chainIds.includes(CHAIN_ID_WITH_A_DEMO)
 
   useEffect(() => {
-    if (!isLoading && safeApp) {
-      trackEvent({
-        ...SAFE_APPS_EVENTS.SHARED_APP_LANDING,
-        label: safeApp.name,
-      })
-      trackEvent({
-        ...SAFE_APPS_EVENTS.SHARED_APP_CHAIN_ID,
-        label: chain.chainId,
-      })
+    if (!isLoading && safeApp?.chainIds.length) {
+      const appName = backendApp ? backendApp.name : safeApp.url
+
+      trackSafeAppEvent(SAFE_APPS_EVENTS.SHARED_APP_LANDING, appName)
```

Comment:
Do we really need 2 separate events for this? Can the chainid be included in the `SHARED_APP_LANDING`?

Reply:
yep, changed it. Does not make sense